probe-rs/src/vendor/nxp/sequences/nxp_armv8m.rs

```diff
@@ -515,16 +516,17 @@ impl MIMXRT5xxS {
             // generalize this so that the reset is configurable?
             //
             // See MIMX685-EVK schematics page 12 for details.
-            interface.write_word_32(0x40021044, 1 << 2)?; // enable HSGPIO2 clock
-            interface.write_word_32(0x40000074, 1 << 2)?; // take HSGPIO2 out of reset
             interface.write_word_32(0x40004130, 0x130)?; // full drive and pullup
+            interface.write_word_32(0x40021044, 1 << 2)?; // enable HSGPIO2 clock
+            interface.write_word_32(0x40020074, 1 << 2)?; // take HSGPIO2 out of reset
```

Comment:
But now you're configuring drive strength and pull-up while the GPIO clock is potentially gated. That's the reason why I swapped the other of these two writes. What happens if you gate HSGPIO2 clock and attempt to modify a GPIO2 register?

Reply:
Having thought about it I agree that your ordering is correct. I'll change it with an extra comment that it differs from the cmsispack implementation.

Reply:
Do we have to adjust the MR description to represent the change correctly now?

Reply:
Just did!